code/examples/unsafe_2.rs

```diff
@@ -1,19 +1,14 @@
 use std::mem::MaybeUninit;
 
 fn main() {
-    // Safety: all bitpatterns are valid for u16
+    // Safety: all bitpatterns are valid for u8
     let random_number: u8 = unsafe { MaybeUninit::uninit().assume_init() };
 
-    let very_random_number = if random_number <= 100 {
```

Comment:
the OOB is intentional to throw readers off-guard

Reply:
Rolled back